crates/base/src/lib.rs

```diff
@@ -1,5 +1,6 @@
 #![feature(core_intrinsics)]
 #![feature(avx512_target_feature)]
+#![feature(stdsimd)]
```

Comment:
I think this feature is removed a few days ago. Let's update toolchain.

Reply:
> I think this feature is removed a few days ago. Let's update toolchain.

We are currently stuck with toolchain 2024-01-14， maybe we can remove it later?